storm-server/src/main/java/org/apache/storm/localizer/LocalizedResource.java

```diff
@@ -144,6 +144,7 @@ private static Path constructBlobWithVersionFileName(Path baseDir, String key, l
 
     static void completelyRemoveUnusedUser(Path localBaseDir, String user) throws IOException {
         Path userFileCacheDir = getLocalUserFileCacheDir(localBaseDir, user);
+        LOG.info("completelyRemoveUnusedUser {} for directory {}", user, userFileCacheDir);
```

Comment:
This method is about deleting `baseDir/supervisor/usercache/user1` directory. It might be better to log that. `userFileCacheDir` is just part of it.